.yarnrc.yml

```diff
@@ -5,7 +5,9 @@ plugins:
     spec: "@yarnpkg/plugin-constraints"
   - path: .yarn/plugins/@yarnpkg/plugin-postinstall.cjs
     spec: "https://raw.githubusercontent.com/gravitywelluk/yarn-plugin-postinstall/master/bundles/%40yarnpkg/plugin-postinstall.js"
-
-yarnPath: .yarn/releases/yarn-3.1.1.cjs
+  - path: .yarn/plugins/@yarnpkg/plugin-interactive-tools.cjs
```

Comment:
Could be dropped. I find this tool very nice to selectively update dependencies interactively just like with `yarn@1`.

Reply:
Sure, no problem. Thanks.